website/docs/specification/DigitalProductPassport.md

```diff
@@ -18,7 +18,7 @@ import Disclaimer from '../\_disclaimer.mdx';
 | ------ | ----- | ------ | ------ | ------| ---|
 | 0.3.0      | 01-07-2024 | Raw |Refactored to build on untp-core  |[@context](https://test.uncefact.org/vocabulary/untp/dpp/dpp-context.jsonld) | [Schema](../../schema/DPP_v0.3.0_schema.json)|
```

Comment:
I would delete this record, because this version was before we changed the way how we deploy different versions and it is not compatible with the latest versions.
```suggestion

```